manifests/init.pp

```diff
@@ -1,15 +1,12 @@
 # Provides utilities for interacting with a TPM
 #
-# @param ima Toggles IMA on or off.
-#
 # @param take_ownership Enable to allow Puppet to take ownership
 #   of the TPM.
 #
 # @author Nick Markowski <nmarkowski@keywcorp.com>
 # @author Nick Miller <nick.miller@onyxpoint.com>
 #
 class tpm (
-  Boolean $ima            = false,
```

Comment:
Instead of removing (which would require a major version bump), leave in but in the comments, note it is deprecated, does nothing, and will be removed at a later date.

Also, use graph of contributors instead of individuals for @author

Reply:
Marked as deprecated with warning and configured to call the ima module; updated the spec tests accordingly.
Linked @author to contributors graph.